src/yunohost/certificate.py

```diff
@@ -810,9 +810,20 @@ def _domain_is_accessible_through_HTTP(ip, domain):
 
 
 def _name_self_CA():
-    cert = crypto.load_certificate(
-        crypto.FILETYPE_PEM, open(SELF_CA_FILE).read())
-    return cert.get_subject().CN
+    ca_conf = os.path.join(SSL_DIR, "openssl.ca.cnf")
```

Comment:
This is what was done before, actually... But for some reason, opi had a setup where he changed the `cacert.pem` with something else. The "true" info is coming from the `openssl.ca.cnf`, from what I investigated (could be wrong, looks like there's some obscure magic going on in openssl).